### PR TITLE
[ADDED] ~minor :sparkles: Resolving with report, documented in README.md

### PR DESCRIPTION
**Pull request by manuman94**
This solves #8 and #19 issues.

Basically the playAudit method now resolves with the Lighthouse report object.

Contact me if you have any doubt.

Thanks!

### CODE REVIEW

**Comment by abhinaba-ghosh** [comment]
@manuman94 thanks for the PR. Would you mind checking the static analysis failure?

**Comment by manuman94** [comment]
Hi, I don't know why is it complaining. 

I don't think that is an unnecessary block, it's a declaration of an anonymous object. Could you explain how can I fix it?

Thanks!

File: README.md [hunk 1]
```diff
@@ -339,6 +339,14 @@ Sample HTML report:
 
 ![screen](./docs/lighthouse_report.png)
 
+playAudit function also provides a promise that resolves with the Lighthouse result object containing the LHR (Lighthouse report in JSON format).
```
Comment (abhinaba-ghosh) [inline README.md:342]: can we make the intention more clear for the user? We can also add the type of object returned.

File: index.d.ts [hunk 2]
```diff
@@ -36,4 +36,4 @@ export interface playwrightLighthouseConfig {
  */
 export function playAudit(
   playwrightLHConfiguration: playwrightLighthouseConfig
-): Promise<void>;
+): Promise<any>;
```
Comment (abhinaba-ghosh) [inline index.d.ts:39]: Can we construct the object type or specifically obtain the returned object type for the lighthouse itself?

File: src/audit.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ let playAudit = async function (auditConfig = {}) {
     ...auditConfig.reports,
   };
 
-  const { errors, results } = await lighthouse({
+  const { comparison, results } = await lighthouse({
```
Comment (abhinaba-ghosh) [inline src/audit.js:55]: just curious, is it a breaking change with compared to the earlier versions and dependent workflows?

File: index.d.ts (hunk 2 above)
Comment (manuman94) [reply index.d.ts:39]: I wanted to, but I saw that there are no exposed types in Lighthouse library:
https://github.com/GoogleChrome/lighthouse/issues/1773

File: src/audit.js (hunk 3 above)
Comment (manuman94) [reply src/audit.js:55]: I think it doesn't break previous implementations. I only changed the way "lighthouse" returns information.

Previously it returned the result of the `compare(thresholds, newValues)` function, now that is sotred in the `comparison` object and results become the report object.

I tested it for positive and negative use cases (with and without errors) and it works nice. (after the last patch hehe)

File: README.md (hunk 1 above)
Comment (manuman94) [reply README.md:342]: I think it is self-explanatory. What would you add?

The important point here is that you can check the report results in the `results` object resolved by the promise returned by playAudit

**Review by abhinaba-ghosh — APPROVED** [review]

**Comment by abhinaba-ghosh** [comment]
The changes are available in `v2.1.0` :tada:

**Comment by manuman94** [comment]
Nice!

Thank you for this awesome library @abhinaba-ghosh

**Pull request merged**